src/online2/online-nnet3-decoding.h

```diff
@@ -0,0 +1,132 @@
+// online2/online-nnet3-decoding.h
+
+// Copyright 2014  Johns Hopkins University (author: Daniel Povey)
```

Comment:
This file is in the wrong directory-- it should be in nnet3, as it relates to nnet3.  Having nnet3/ depend on nnet2/ is undesirable, but having nnet2/ depend on nnet3/ is really weird.